sigstore-java/src/main/java/dev/sigstore/tuf/TufClient.java

```diff
@@ -206,17 +210,44 @@ void verifyDelegate(
     }
   }
 
-  public void updateTimestamp() {
-    // 1) download the timestamp.json bytes up to few 10s of K max.
+  public void updateTimestamp()
```

Comment:
@asraa or @znewman01 wanna take a quick look over this tuf workflow ?

Reply:
in transit right now but will TAL in then next day or so

Reply:
I'll take a look this evening!

Reply:
func looks great to me!

Reply:
i trust asra, listen to her